src/lib/datasheets/voting.ts

```diff
@@ -50,6 +53,19 @@ export function getHighlightedVoteByGroups(
 		.filter(({ count }) => count > 0);
 }
 
+export function toVoteCardVoting(
+	voting: Voting,
+	highlightedVoteByGroups: HighlightedVoteByGroup[]
+): VoteCardVoting {
+	return {
+		id: voting.id,
+		date: voting.date,
+		nickname: voting.nickname,
+		result: voting.result,
+		votesByGroup: highlightedVoteByGroups
+	};
+}
```

Comment:
This is kind of a temporary adapter function which might be removed in the near future when we change the data source, I suggest you put it in `src/lib/model-component-adapters` as proposed in #169.